www/templates/control/main-control.html

```diff
@@ -5,6 +5,9 @@
       <div ng-click="logOut()" id ="switch-user" class="control-icon-button"><i class="icon ion-log-out"></i></div>
     </div>
     <enketo-demographics-button></enketo-demographics-button>
+    <!-- here is the html related to my (failing) attempts to convert more of the profile screen -->
+    <setting-row text-key = "control.view-privacy" icon-name = "eye"></setting-row>
```

Comment:
```suggestion
    <setting-row text-key="'control.view-privacy'" icon-name="'eye'"></setting-row>
```

@Abby-Wheelis Put single quotes inside the double quotes and the strings will pass in like you want them to.
I believe this AngularJS thing, not a React thing.